library/build.gradle.kts

```
@@ -70,6 +70,10 @@ android {
     buildFeatures {
         buildConfig = true
     }
+    defaultConfig {
+        val consumerFile = file("src/androidMani/consumer-rules.pro")
```

```suggestion
        val consumerFile = file("src/androidMain/consumer-rules.pro")
```

So we want to have that file in android? 🤔
We can also just keep it here in library... 💭